src/pages/Beauty.jsx

```diff
@@ -1,7 +1,17 @@
+import styled from "styled-components";
+import FooterNavbar from "../components/FooterNavbar";
+import Header from "../components/Header";
+import beautyheader from "../images/Beauty.png";
+import { Routes, Route, NavLink, Link } from "react-router-dom";
+
 function Beauty() {
   return (
     <>
-      <h2>Beauty</h2>
+      <Link to="/">
+        <Header image={beautyheader} altText="Beauty Header" />
+      </Link>
```

Comment:
Since all headers lead to the index page, you could move the `Link` inside the Header component.

Reply:
Thank you very much for your code-review! :)
I've changed the code according to your suggestion. Please review my code once again. I'm looking forward to your approval :)